src/webgl/p5.Geometry.js

```diff
@@ -238,12 +240,16 @@ p5.Geometry.prototype._makeTriangleEdges = function() {
  * @chainable
  */
 p5.Geometry.prototype._edgesToVertices = function() {
+  const data = this.lineVertexColors.slice();
```

Comment:
Do we need to make a copy of `lineVertexColors` or does it not work if we reference it directly? If we do need to copy it, maybe we can call this something more descriptive, like `lineColorData`

Reply:
Here, lineVertices are newly created from vertices, but since the color arrangement of lineVertexColors corresponds to vertices, it does not correspond to lineVertices as it is.
So, after making a copy of lineVertexColors corresponding to vertices in advance, we use that data to create an array of colors corresponding to lineVertices. So I thought I should make a copy.
As for the name, lineColorData is certainly easier to understand, so I would like to do so!

Reply:
It makes sense that `lineVertexColors` is built using info in `lineColorData` to correspond to the data in `lineVertices`. Does the code work if we do `const lineColorData = this.lineVertexColors;`? So, replacing `this.lineVertexColors.slice()` with `lineVertexColors`? Having a separate variable for it is fine if that communicates the intention of the variable more clearly, but I'm just trying to avoid the performance cost of doing an extra copy if we can.

Reply:
After thinking about it overnight, I realized I didn't need to make a copy, so I'd like to rewrite it! I'm sorry...

Reply:
No need to apologize! that's what code review is for :D